rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -261,6 +260,22 @@ public List<ModuleDescription> getRpcModules() {
             String name = configElement.getString("name");
             String version = configElement.getString("version");
             boolean enabled = configElement.getBoolean("enabled");
+            int timeout = 0;
+            Map<String, Integer> methodTimeoutMap = new HashMap<>();
+
+            if (configElement.hasPath("timeout")) {
+                timeout = configElement.getInt("timeout");
+            }
+
+            if (configElement.hasPath("methodTimeout")) {
+                configElement.getObject("methodTimeout")
+                        .unwrapped()
+                        .entrySet()
+                        .stream()
+                        .map(entry -> new AbstractMap.SimpleEntry<>(entry.getKey(), (Integer) entry.getValue()))
+                        .forEach(entry -> methodTimeoutMap.put(entry.getKey(), entry.getValue()));
+            }
```

Comment:
Minor thing, but I would extract this block to a helper method, this loop es already long enough in lines.